README.md

```diff
@@ -60,6 +60,7 @@ The plugin comes with vast set of configuration options.
 | `WP_REDIS_DISABLE_METRICS`           | `false`     | Disables metrics collection and display |
 | `WP_REDIS_DISABLE_BANNERS`           | `false`     | Disables promotional banners |
 | `WP_REDIS_DISABLE_DROPIN_AUTOUPDATE` | `false`     | Disables the drop-in auto-update |
+| `WP_REDIS_TLS`                       |             | TLS connection options for `tls` or `rediss` scheme |
```

Comment:
Let's make the default an empty array `[]` and link to https://www.php.net/manual/en/context.ssl.php in the description.

Reply:
Done